pkg/apis/forklift/v1beta1/plan.go

```diff
@@ -106,6 +106,13 @@ type PlanSpec struct {
 	// Determines if the plan should migrate shared disks.
 	// +kubebuilder:default:=true
 	MigrateSharedDisks bool `json:"migrateSharedDisks,omitempty"`
+	// Determines if the guest conversion pod should be deleted after successful migration.
```

Comment:
```suggestion
	// DeleteGuestConversionPod determines if the guest conversion pod should be deleted after successful migration.
```

need to start with the name